config/api_router.py

```diff
@@ -15,5 +20,6 @@
 router.register("workflows", WorkflowViewSet, basename="workflows")
 router.register("workflow-update", WorkflowPartialUpdateViewSet, basename="workflow-update")
 router.register("workflow-ticket", WorkflowTicketViewSet, basename="workflow-ticket"),
+router.register("author", AuthorWorkflowSubmissionViewSet, basename="author-submission"),
```

Comment:
```suggestion
router.register("authors", AuthorWorkflowSubmissionViewSet, basename="author-submission"),
```

https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-name-collections-with-plural-nouns